### ISSUE TITLE
BZD: misclassify brain zone

### ISSUE THREAD

**Issue by yuanci89** [mnarizzano/SEEGA#115]
The electrodes are not getting placed in the right anatomic location. I used the Destrieux atlas, and saved it as a label map. However, it's listing contact points all in the wrong spots. For example, LA1.0 as in the 3rd ventricle even though it's clearly in the left amygdala. the label map and markup point list for the contact points are co-registered in the same space. Am I missing a step?

My contact point list, destrieux label map, and t1 mprage can be found here: https://www.dropbox.com/scl/fi/8bkfvockzw896zde6of9c/brain-zone-misclassified.zip?rlkey=ox8od7vqyobwjkz9u8pp17lgh&st=7fzzv0mn&dl=0

Thank you for all the help!

![image](https://github.com/user-attachments/assets/94ad10f6-a79b-49c8-b8e3-225d26b147bb)

**Comment by gabri470** [comment]
Dear @yuanci89 thanks a lot for sharing the data. I gonna look into this asap and report back to you.

**Comment by yuanci89** [comment]
@gabri470 Thank you so much! I'm new to this whole thing so I'm trying my best to figure things out on my own and not post so much but I feel like I am posting a lot :)

**Comment by yuanci89** [comment]
@gabri470 I believe the issue is the only option for look up table is "FreeSurferColorLUT20120827" which does not line up with my labelmap. So I tried loading the .mgz file from the freesurfer output directly, however it still gives me the same result... I added the .mgz file to that dropbox link (from the original post) so when you download the zip file it will also have that .mgz file.

Is there a way to modify the color look up table to be a custom one so it's giving me the correct brain zones?
